FluentAvalonia/Styling/Core/FluentAvaloniaTheme.axaml

```diff
@@ -0,0 +1,16 @@
+﻿<Styles
+    xmlns="https://github.com/avaloniaui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    x:Class="FluentAvalonia.Styling.FluentAvaloniaTheme">
+    <Styles.Resources>
+        <ResourceDictionary>
+            <FontFamily x:Key="SymbolThemeFontFamily">/Fonts/#Symbols</FontFamily>
+            <Color x:Key="SystemAccentColor">#00000000</Color>
```

Comment:
A placeholder, SystemAccentColor will be override at the runtime.

Reply:
I can't repro that exception so let's take this out. It's odd that's the one it was tripping on though as most of fluent v2 uses one of the light or dark variants rather than the straight accent color